packages/codec/lib/abi-data/allocate/index.ts

```diff
@@ -605,13 +670,17 @@ function getCalldataAllocationsForContract(
   abi: AbiData.Abi,
   contractNode: Ast.AstNode,
   constructorContext: Contexts.DecoderContext,
+  deployedContext: Contexts.DecoderContext,
   referenceDeclarations: Ast.AstNodes,
   userDefinedTypes: Format.Types.TypesById,
   abiAllocations: AbiAllocations,
   compiler: Compiler.CompilerVersion
 ): CalldataAllocationTemporary {
   let allocations: CalldataAllocationTemporary = {
-    constructorAllocation: defaultConstructorAllocation(constructorContext), //will be overridden if abi has a constructor
+    constructorAllocation: defaultConstructorAllocation(
+      constructorContext,
+      deployedContext
```

Comment:
Does it make sense to group these contexts, similarly to how you have `CalldataAndReturndataAllocation` grouped?

Reply:
I don't think so.  But rather than explain why, I'll just note that uhhh I only now realized that I left that `deployedContext` argument in, even though the final version doesn't use it (I added it when I was trying to keep bytecode unified with the rest, which as you know I ultimately gave up on).  So, uh, that argument should be removed, is what I should actually do.